ts/store/reducers/allowedSnapshotScreens.ts

```diff
@@ -3,10 +3,10 @@ import ROUTES from "../../navigation/routes";
 import { isDebugModeEnabledSelector } from "./debug";
 import { currentRouteSelector } from "./navigation";
 
-export const screenBlackList = new Set([ROUTES.WALLET_ADD_CARD]);
+export const screenBlackList = new Set([ROUTES.WALLET_ADD_CARD as string]);
 
 /**
- * Return {true} if the current screen can be snapshotted (android only).
+ * Return {true} if a snapshot can be made in the current screen (android only).
```

Comment:
🤓 **suggestion**: we normally _take_ a snaphot rather than _make_ one -  I mean, since you corrected it.... 😸

Reply:
You're right, thanks! Updated in [79b1989](https://github.com/pagopa/io-app/pull/3756/commits/79b1989d030390fb8e00ff993a9c86d123b57411)